_includes/footer.html

```diff
@@ -15,6 +15,9 @@
         <nav>
           <ul class="site-social-links inline">
               <li><a target="_blank" title="RSS Feed" href="/feed.xml" class="ion-social-rss"></a></li>
+              {% if site.guardian.social_links.email_subscription_url %}
```

Comment:
Wait, so this just links to widen.com? As I understand it, is the plan to change it in the future to the Feedburner URL?

Reply:
This is an example for local development in the theme. When implementing in Developer Blog, the actual value of this config will be: https://feedburner.google.com/fb/a/mailverify?uri=WidenDeveloper

If this proves useful, we can also setup a configuration for the Engineering Blog and change the value within that blog as well. I did look into things like MailChimp as well, which would also work here so this work is not tightly coupled to any specific email subscription.

Reply:
Oh right, I forgot this was the root template. 👍